frontend/src/components/dashboard/DashboardHeader.jsx

```diff
@@ -53,16 +48,38 @@ export default class DashboardHeader extends React.Component {
     this.setState({ searchValue: event.target.value });
   };
 
+  componentDidMount() {
+    this.regularFetchInfo();
+  }
+
+  componentWillUnmount() {
+    clearTimeout(this.timer);
+    this.timer = false;
+  }
+
+  fetchInfo = async () => {
+    const details = await new DetailsApi().getDetails().catch(() => null);
+    this.setState({ lastBlockHeight: details.lastBlockHeight });
```

Comment:
Why do we ignore the rest information?

Reply:
the auto refresh is only for block height

Reply:
It seems to be completely fine to update the rest of the information as well. What do you think? Does it break user expectations in some way in your opinion?

Reply:
No, let's update all since we fetch all information